lib/rubocop/cop/minitest/test_file_name.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks if test file names start with `test_` or end with `_test.rb`.
```

Comment:
I think it can be mentioned that files containing classes with names ending in `Test` are considered test files.

Reply:
Seeing https://github.com/rubocop/minitest-style-guide/pull/52, I thought it would also mention the risk of tests not running if naming is not followed.